src/aap_eda/settings/default.py

```diff
@@ -83,6 +83,16 @@
                              "target": "/var/run/containers/storage",
                              "type": "bind"}]'
 
+
+Django Ansible Base settings:
+For Resource Server the following are required when
+DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED is turned on:
+* RESOURCE_SERVER__URL - The URL to connect to the resource server
+* RESOURCE_SERVER__SECRET_KEY - The secrect key to connect to the server
```

Comment:
Typo:
```suggestion
* RESOURCE_SERVER__SECRET_KEY - The secret key to connect to the server
```

I think the description may be incorrect. I guess we don't need the secret key to perform the connection but to be able to encrypt/decrypt data crossed with the resource server. Am I wrong?